linera-client/src/client_context.rs

```diff
@@ -477,13 +480,13 @@ where
         let default_chain_id = self
             .wallet
             .default_chain()
-            .context("should have default chain")?;
+            .expect("should have default chain");
```

Comment:
These are `expect`s rather than having dedicated error variants because they are only present in the benchmarking code, which we set up rather particularly, and otherwise aren't generally considered error conditions.

Reply:
Got it, thanks!